Parse/src/main/java/com/parse/ParseRESTObjectBatchCommand.java

```diff
@@ -63,7 +63,7 @@
       for (ParseRESTObjectCommand command : commands) {
         JSONObject requestParameters = new JSONObject();
         requestParameters.put("method", command.method.toString());
-        requestParameters.put("path", String.format("/1/%s", command.httpPath));
+        requestParameters.put("path", String.format("%s/%s", server.getPath(), command.httpPath));
```

Comment:
Trailing slash issue would still appear in batch encoding, I don't have a great solution for that however, ATM.